src/ticks/debugger.c

```diff
@@ -13,6 +13,10 @@
 #include <stdarg.h>
 #endif
 
+#ifdef _MSC_VER
```

Comment:
@suborb

Reply:
oh wait. I am dumb

Reply:
@suborb I'm actually a little confused. Do I need winsock for mingw or should I change the debugger_init def to be msc only also?

Reply:
Winsock is needed for all toolchains on windows.

Reply:
I think that means this line: https://github.com/z88dk/z88dk/blob/master/src/ticks/Makefile#L27 needs the now badly named variable `$(GDBLDFLAGS)` adding to the end of it - see the rule for z88dk-gdb